packages/server/api/src/app/ai/chat/ai-chat.controller.ts

```diff
@@ -53,6 +57,12 @@ export const aiChatController: FastifyPluginAsyncTypebox = async (app) => {
         await createChatContext(chatId, chatContext);
       }
 
+      sendAiChatMessageOpenEvent({
```

Comment:
I added two events, i wasnt sure if it would be interesting to see people open the chat without sending a message